Proton/Sniffs/Commenting/InlineCommentSniff.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Proton\Sniffs\Commenting;
+
+use PHP_CodeSniffer\Files\File;
+use PHP_CodeSniffer\Sniffs\Sniff;
+
+class InlineCommentSniff implements Sniff
+{
+    /**
+     * @inheritDoc
+     */
+    public function register(): array
+    {
+        return [T_COMMENT];
+    }
+
+    /**
+     * @inheritDoc
+     */
+    public function process(File $phpcsFile, $stackPtr): void
+    {
+        $tokens = $phpcsFile->getTokens();
+
+        if ($tokens[$stackPtr]['content'][0] === '#') {
+            $phpcsFile->recordMetric($stackPtr, 'Inline comment style', '# ...');
```

Comment:
I would make this sniff doing one thing and make it work with `#`, `//` or `/*`, we can use another sniff to disallow perl style comments

Reply:
You are right - I pushed an update so we keep the separation of concerns. I readded the `PEAR.Commenting.InlineComment` rule that takes care of disallowing pearl-style comments.